godtools/App/Services/Renderer/Views/MobileContent/Views/Pages/MobileContentPagesViewModel.swift

```diff
@@ -97,7 +97,7 @@ class MobileContentPagesViewModel: NSObject, MobileContentPagesViewModelType {
             return
         }
         
-        translationsRepository.getTranslationManifestsFromRemote(translations: translationsNeededDownloading, manifestParserType: .renderer, includeRelatedFiles: true)
+        translationsRepository.getTranslationManifestsFromRemote(translations: translationsNeededDownloading, manifestParserType: .renderer, includeRelatedFiles: true, shouldFallbackToLatestDownloadedTranslationIfRemoteFails: false)
```

Comment:
Hey @rachaelblue there is only 1 case currently where that is set to false which is when a tool is currently being viewed and downloading the latest version fails.  In that case a user is already viewing the latest downloaded version.

I just realized I need to update some of the logic in here for downloading the translation zip file.  If a network requests fails due to being offline or is cancelled then zip downloading should be skipped.

Reply:
Hey @rachaelblue just made an update, the logic in there is getting a little messy, but this is the logic.
If downloading translation using new distribution api's fails then a couple things happen.
1. If the request was cancelled we exit.
2. If the request failed and it wasn't because of a notConnectedToInternet, then attempt to download and cache the translation zip file.  Here if downloading the zip fails then attempt to fallback to the latest downloaded translation version. 
3. If the original request failed due to notConnectedToInternet then attempt to fallback to the latest downloaded translation version.

Reply:
At the moment there isn't a good way to test.  We will need Ann to test by doing the following.

1. Connect device to network. Open a Tool Translation this way it is marked as downloaded.
2. Update the Tool Translation in OneSky.
3. Pull to refresh GodTools or relaunch from a killed state so updated Tool Translation is added to the Realm Database.
4. Set device to Airplane mode and open the updated Tool Translation.  In this case it should attempt to download the latest version and then fallback to the downloaded Tool Translation in step 1.